data-prepper-plugins/geoip-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/extension/databasedownload/DBSource.java

```diff
@@ -25,11 +25,8 @@
 
 public interface DBSource {
 
-    public static final Logger LOG = LoggerFactory.getLogger(DBSource.class);
-    public String tempFolderPath = System.getProperty("java.io.tmpdir")+ File.separator +"GeoIP";
-    public String tarFolderPath = tempFolderPath + "/tar";
-    public String downloadTarFilepath = tarFolderPath + "/out.tar.gz";
-    public void initiateDownload(List<String> config) throws Exception;
+    Logger LOG = LoggerFactory.getLogger(DBSource.class);
```

Comment:
Does this need to be at interface level?

Reply:
I will try to refactor this. There is a LOG in line 44, I will try to remove this.

Reply:
Loggers in interfaces is probably unnecessary.